### ISSUE TITLE
Update documentation for development practices

### ISSUE THREAD

**Issue by fjbo-net** [cotheca/docs#30]
## Description
A clear and concise description of what the problem is or how is this feature missing.

### Objectives
Please specify what the finished feature should accomplish (one or more objectives):
- [x] Create a new feature branch 'feature-30-UpdateDocumentationForDevelopmentPractices' for this issue
- [x] Explain how to do things with the new methodology and how views are relevant for different tasks.
  - [x] Update existing documentation on sub-branches
    - [x] (1.) #33

    - [x] (2.) Pull changes from 'main' after updating information into 'feature-30-UpdateDocumentationForDevelopmentPractices'

    - (3.) For each existing branch with references to the development process:
      - Create a new issue, specific to each branch
      - Create new sub-branch for each existing branch
      - Sync changes from 'main'
      - Resolve any conflicts to existing changes.
      - Remove any references to the old ways of doing things
      - Explain how to do things with the new methodology and how views are relevant for different tasks
      - Merge back into the existing branch
      - [x] #38
        - Resolve any conflicts with existing changes
        - Remove any references to old information about:
          - Development cycle
          - Boards
          - Scrum
          - Kanban

**Comment by fjbo-net** [comment]
Blocks:
- #27

**Comment by fjbo-net** [comment]
Restructured objectives. No longer a blocker for:
- #27

**Comment by fjbo-net** [comment]
No longer blocked by:
- #33

**Comment by fjbo-net** [comment]
Updated item size since most tasks have been completed and it's pretty likely to be finished before the end of this sprint.

**Comment by fjbo-net** [comment]
Currently there is 3 active branches:
- [feature-30-UpdateDocumentationForDevelopmentPractices](https://github.com/cotheca/docs/tree/feature-30-UpdateDocumentationForDevelopmentPractices)
  - Issue's branch
- [wip-7-SeparationOfConcerns ](https://github.com/cotheca/docs/tree/wip-7-SeparationOfConcerns)
  - Needs to be synced with 'main' given that it contains development-related files or documents.
- [wip-1-InitialPractices](https://github.com/cotheca/docs/tree/wip-1-InitialPractices)
  - Does not require to be synced

**Comment by fjbo-net** [comment]
Blocked by:
- #38

**Comment by fjbo-net** [comment]
No longer blocked by:
- #38
